### ISSUE TITLE
PFR does not work with non movable types

### ISSUE THREAD

**Issue by apolukhin** [boostorg/pfr#68]

**Comment by denzor200** [comment]
What do you mean by "non movable types"?
In my artificial example it's right

```
struct A
{
    A() = default;
    std::string type;
    std::string value;

    A(const A&) = default;
    A& operator=(const A&) = default;

    A(A&&) = delete;
    A& operator=(A&&) = delete;
};

A a;
A b = a;
// A c = std::move(a); ///< error: use of deleted function ‘A::A(A&&)’

boost::pfr::get<0>(a) = "Some type";
boost::pfr::get<1>(a) = "Hello world!!";

std::cout << boost::pfr::get<0>(a) << " : " << boost::pfr::get<1>(a) << std::endl; ///< outputs "Some type : Hello world!!"
```
Or are you not talking about that?

**Comment by apolukhin** [comment]
The copy constructor should be also deleted https://github.com/boostorg/pfr/blob/develop/test/run/non_movable.cpp

The issue was fixed in https://github.com/boostorg/pfr/commit/638fefedd09256b928f05e2aaf11f0a941df4b22#diff-99686eb583b72a779722f606ed8a9a5322d3a34eaf637e2f8ae0ed1c6f7eac11 and https://github.com/boostorg/pfr/commit/f33c357af6f74c319a0277993e16141ac40b00ff

Will be merged to the master after the 1.76 release

**Comment by apolukhin** [comment]
Merged to master
